### ISSUE TITLE
MAS 2.5.3 - Label in Name

### ISSUE THREAD

**Issue by scottbommarito** [NuGet/NuGetGallery#7532]
UI components with labels that include text or images of text should have names that contain the text that is presented visually. E.g. a button with the visible text "submit" must have an `aria-label` that contains "submit" such as "submit the form".

I believe we are already compliant with this, but there may be some controls on the site that we need to fix.

Terminology:
- labels are the visible text used to identify control to all users
- names are the programmatic or accessible name of a control for screen readers/assistive technology

**Comment by scottbommarito** [comment]
Potential violations:
- Manage page: Listing and Documentation pages have "Select version" label with an `aria-label` of "Select a version"
- "many many more" link on About page has `aria-label` of "Read the full list of contributors on GitHub."
- "FAQ" link in footer has `aria-label` of "Frequently Asked Questions"
- "Register new" certificate button has `aria-label` of "Register a new certificate"
- Search bar has `aria-label` of "Enter packages to search" and a `placeholder` of "Search for packages..."
- Search bar search button has an `aria-label` of "Search" and a `title` of "Search for packages"
- Breadcrumbs for managing my organizations/managing this organization on profile page have inconsistent text/labels
